### PR TITLE
Imu docs

### PR DESCRIPTION
**Pull request by jgueldenstein**
## Necessary checks
- [x] Run `catkin build`
- [x] Write documentation
- [x] Create issues for future work #76
- [x] Test on your machine
- [x] Put the PR on our Project board

### CODE REVIEW

**Review by SammyRamone — APPROVED** [review]

**Comment by SammyRamone** [comment]
maybe add a link to this docs in the IMU git?

File: bitbots_docs/docs/manual/hardware/imu_dxl.rst [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+
+
+.. _Accelerometer Calibration:
+
+Accelerometer Calibration
+=========================
+
+It is necessary to calibrate the accelerometer once before using it.
+For this the accelerometer must be placed with the x,y, and z-axis pointing downwards and upwards once.
+We have designed the 3D printed case for the board in such a way that this is relatively easy.
```
Comment (jaagut) [inline bitbots_docs/docs/manual/hardware/imu_dxl.rst:291]:
```suggestion
For this, the accelerometer must be placed with the x,y, and z-axis pointing downwards and upwards once.
We have designed the 3D printed case for the board in such a way, that this is relatively easy.
```

File: bitbots_docs/docs/manual/hardware/imu_dxl.rst [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+
+Before starting the calibration, you should check the accelerometer measurements.
+For each of the axes pointing downwards or upwards the value should be at least 7.5 m/s^2.
+If this is not the case, you need to lower the **accel_calibration_threshold**.
+
+To perform the calibration procedure follow this procedure:
+
+1. Place the IMU on one of the 6 sides
+2. Set a 1 to the **calibrate_accel** register (rosservice call /imu/calibrate_accel if you are using our software)
```
Comment (jaagut) [inline bitbots_docs/docs/manual/hardware/imu_dxl.rst:300]:
```suggestion
2. Set a 1 to the **calibrate_accel** register (rosservice call ``/imu/calibrate_accel``, if you are using our software)
```

File: bitbots_docs/docs/manual/hardware/imu_dxl.rst [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+3. Wait until the IMU responds to reads again (or around 5 seconds)
+4. repeat for remaining 5 sides
```
Comment (jaagut) [inline bitbots_docs/docs/manual/hardware/imu_dxl.rst:302]:
```suggestion
4. Repeat for remaining 5 sides
```

File: bitbots_docs/docs/manual/hardware/imu_dxl.rst [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+
+After the procedure you should check the values in the **accel_scale** and **accel_bias** registers.
```
Comment (jaagut) [inline bitbots_docs/docs/manual/hardware/imu_dxl.rst:304]:
```suggestion
After the procedure, you should check the values in the **accel_scale** and **accel_bias** registers.
```

**Review by jaagut — APPROVED** [review]

**Comment by jaagut** [comment]
All suggestions will be merged with #82.

**Pull request merged**